hi-utils/src/com/hh/redis/RedisUtils.java

```diff
@@ -0,0 +1,275 @@
+package com.hh.redis;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author TruongNX25
+ */
+
+import org.apache.log4j.Logger;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+
+public class RedisUtils {
+
+    private RedisUtils() {
+    }
+
+    protected static Map<String, JedisPool> redisMap = new HashMap<String, JedisPool>();
+    private static Logger log = Logger.getLogger(RedisUtils.class.getName());
+    // Let's use JedisPool for thread-safe xD
+    protected static final String HOST_REDIS_DEFAULT;
+    protected static final int PORT_REDIS_DEFAULT;
+
+    static {
+        // HOST_REDIS_DEFAULT = "10.58.244.172";
+        HOST_REDIS_DEFAULT = "bi172";
```

Comment:
Consider removing it, or change it to localhost, cause it contains confidential information.